test/test.config

```diff
@@ -23,10 +23,9 @@
        {get_options, [{r, 2}, {pr, 0}]}
      ]}
    ]},
-
    {docs, [
-     {sumo_test_people_riak, sumo_test_riak},
-     {sumo_test_purchase_order, sumo_test_riak}
+     {people, sumo_test_riak, #{module => sumo_test_people_riak}},
+     {purchase, sumo_test_riak, #{module => sumo_test_purchase_order}}
```

Comment:
Ok I see, I got your point, in that case you're right, I agree it should be named `purchases`